app/configuration.py

```diff
@@ -179,17 +187,24 @@ def _validate_config(config: dict):
                     item in config[req_key] and config[req_key][item]
                 ), f"No {item!r} provided in {req_key!r} section of configuration"
 
-        def _is_valid_command_topic(topic):
+        def _is_valid_mqtt_topic(topic):
             # Command topic must be str, must not have more than 7 levels
-            return (
-                isinstance(config["mqtt_settings"]["command_topic"], str)
-                and config["mqtt_settings"]["command_topic"].count("/") <= 7  # noqa
-            )
+            return isinstance(topic, str) and topic.count("/") <= 7
 
-        assert _is_valid_command_topic(
+        assert _is_valid_mqtt_topic(
             config["mqtt_settings"]["command_topic"]
         ), "The command topic must be a valid MQTT topic name"
 
+        error_topic = config["mqtt_settings"].get("error_topic")
+        if error_topic:
+            assert _is_valid_mqtt_topic(
+                error_topic
+            ), "The error topic must be a valid MQTT topic name"
+
+            assert (
```

Comment:
oh!! you are doing that here. Awesome
You are missing the `+` symbol though

Reply:
thanks, I've added that